packages/realm-web/README.md

```diff
@@ -4,15 +4,13 @@ Accessing MongoDB Realm from a web-browser.
 
 ## Installation
 
-In its current form, the package has not been published to NPM, instead its distributed using a direct HTTP URL:
-
 ```
-npm install https://static.realm.io/downloads/realm-web/realm-web-0.1.0.tgz
+npm install realm-web --save
```

Comment:
`--save` has been the default behaviour since `v5.0.0`.

Reply:
Interesting ...